newapi/ooniapi/citizenlab.py

```diff
@@ -589,3 +590,221 @@ def post_propose_changes():
     ulm = get_url_list_manager()
     pr_id = ulm.propose_changes(username)
     return pr_id
+
+
+# # Prioritization management # #
+
+
+def create_url_priorities_table() -> None:
+    log = current_app.logger
+    sql = "SELECT to_regclass('url_priorities')"
+    q = current_app.db_session.execute(sql)
+    if q.fetchone()[0] is not None:
+        return  # table already present
+
+    log.info("Creating table url_priorities")
+    sql = """
+CREATE TABLE public.url_priorities (
+    category_code text,
+    cc text,
+    domain text,
+    url text,
+    priority smallint NOT NULL,
+    UNIQUE (category_code, cc, domain, url)
+);
+COMMENT ON COLUMN public.url_priorities.domain IS 'FQDN or ipaddr without http and port number';
+COMMENT ON COLUMN public.url_priorities.category_code IS 'Category from Citizen Lab';
+GRANT SELECT ON TABLE public.url_priorities TO readonly;
+GRANT SELECT ON TABLE public.url_priorities TO shovel;
+GRANT SELECT ON TABLE public.url_priorities TO amsapi;
+"""
+    current_app.db_session.execute(sql)
+
+    log.info("Populating table url_priorities")
+    sql = """INSERT INTO url_priorities
+        (category_code, cc, domain, url, priority)
+        VALUES(:category_code, '*', '*', '*', :priority)"""
+    category_priorities = {
+        "NEWS": 100,
+        "POLR": 100,
+        "HUMR": 100,
+        "LGBT": 100,
+        "ANON": 100,
+        "GRP": 80,
```

Comment:
We want this to be more highly prioritised than NEW. We should bump it up to 150.